website/content/docs/overview/other-software.mdx

```diff
@@ -0,0 +1,134 @@
+---
+layout: docs
+page_title: Boundary vs. Other Software
+description: Compares Boundary to other security technologies
+---
+
+# Boundary vs. Other Software
+
+Boundary is a software solution that provides secure remote access to infrastructure services.
+It can be difficult to understand how to compare the different remote access security solutions that are available on the market.
+Is one a replacement for another?
+Are they complementary?
+In this topic, we compare Boundary to other available security solutions.
+
+## Introduction
+
+The following comparisons are philosophical in nature, and are not intended to be opinionated.
+There are many possible remote access use cases, and the following solutions could have some part in your overall security strategy.
```

Comment:
I added this in response to Jairo's comment from the doc about removing the word "negative." But I'm not sure if "opinionated" is the best word. I am open to any suggestions. Thanks!

Reply:
Perhaps just `The following comparisons are philosophical in nature, and are not intended to be oppositional.`

Reply:
> Perhaps just `The following comparisons are philosophical in nature, and are not intended to be oppositional.`

Nice, I like that.